sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -327,19 +387,23 @@ def __repr__(self):
 
 
 class RecognizeLinkedEntitiesResult(DictMixin):
-    """RecognizeLinkedEntitiesResult.
+    """RecognizeLinkedEntitiesResult is a result object which contains
```

Comment:
Consider changing to something like:
 ... which contains links to a well-known knowledge base, like for example,Wikipedia or Bing.